src/core/hle/D3D8/Direct3D9/Direct3D9.cpp

```diff
@@ -3285,6 +3285,7 @@ LTCG_DECL xbox::dword_xt* CxbxrImpl_MakeSpace(void)
 // which is normally the entry point for all Xbox D3D 'things'.
 void HLE_PushInit()
 {
+	SwitchToThread();
```

Comment:
Please add a comment explaining succinctly (but accurately!) why this is apparently needed (and what alternatives exist but are not suitable).

Without such a comment there's bound to be confusion and high risk of removal later on

Reply:
fixed, hopefully it is clear